docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Berlin
+
+RUN apt update \
+    && apt-get install -y \
+    g++ \
+    cmake \
+    libatlas-base-dev \
+    python3 \
+    python3-dev \
+    python3-pip \
+    swig \
+    git\
+    libhdf5-serial-dev\
+    && ln -sf /usr/bin/swig4.0 /usr/bin/swig
+    
+RUN pip3 install python-libsbml>=5.17.0
+
+COPY amici.tar.gz /tmp
+
+ENV AMICI_CXXFLAGS -fopenmp
+ENV AMICI_LDFLAGS -fopenmp
+
+RUN pip3 install -U --upgrade pip wheel \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=5388382)